_includes/header.html

```diff
@@ -55,4 +55,4 @@
   </div>
 </nav>
 
-<span id="forkongithub"><a alt="Fork me on GitHub" aria-label="Fork me on GitHub" href="{{site.repo_url}}/tree/main/{{page.path}}" target="_blank">Fork me on GitHub</a></span>
```

Comment:
was this not working? The thinking behind was it was doing was to link directly to the page that you'd want to make the change to, but I guess that doesn't have to be the case for this particular header item...

Reply:
It was working. I was thinking the user would want to go to the main page. But I can change it back if you'd prefer.

Reply:
Nah, I trust your instincts here. That said, you should be able to use `href="{{site.repo_url}}"` if you didn't want to hardcode the path in there.

Reply:
Good point. I updated that href via the GH interface, as VS Code has stopped working for some reason.